sqlglot/expressions.py

```diff
@@ -1641,6 +1683,11 @@ def join(
             on = and_(*ensure_list(on), dialect=dialect, **opts)
             join.set("on", on)
 
+        if using:
+            if isinstance(using, str):
+                using = [to_identifier(name) for name in using.split(", ")]
```

Comment:
Why are we accepting a CSV instead of a list of strings?

Reply:
Right, I'll fix this too. Didn't notice that's what we were doing for `on` due to docstring type signature confusion. Thanks!

Reply:
you can call ensure_list